src/app/modules/item/services/item-task.service.ts

```diff
@@ -68,12 +70,16 @@ export class ItemTaskService {
   }
 
   private bindPlatform(task: Task): void {
+    if (!this.attemptId) throw new Error('attemptId must be defined');
```

Comment:
For easier debugging, add  "... The "configure" has probably not been called as expected."